dolphinscheduler-e2e/dolphinscheduler-e2e-core/src/main/java/org/apache/dolphinscheduler/e2e/core/DolphinSchedulerExtension.java

```diff
@@ -73,6 +74,7 @@
     @Override
     @SuppressWarnings("UnstableApiUsage")
     public void beforeAll(ExtensionContext context) throws IOException {
+        Testcontainers.exposeHostPorts(8888);
```

Comment:
this will start a new container, let's put this line to `if (LOCAL_MODE) {` so it's not started when unnecessary.